### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by simonschaufi**

### CODE REVIEW

**Comment by simonschaufi** [comment]
@kevinpapst any reason why this takes so long to merge?

File: _documentation/emails.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ You have to adapt two settings in your `.env` [configuration file]({% link _docu
 
 ### MAILER_URL
 
-The following list of providers are supported out-of-the-box by Kimai: 
+The following list of providers is supported out-of-the-box by Kimai: 
 
 - Deactivated: `null://null`
 - SMTP: `smtp://localhost:25?encryption=&auth_mode=`
```
Comment (kevinpapst) [inline _documentation/emails.md:26]: Outdated example
```suggestion
- SMTP: `smtp://localhost:25`
```

File: _documentation/emails.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ You have to adapt two settings in your `.env` [configuration file]({% link _docu
 
 ### MAILER_URL
 
-The following list of providers are supported out-of-the-box by Kimai: 
+The following list of providers is supported out-of-the-box by Kimai:
```
Comment (kevinpapst) [inline _documentation/emails.md:23]:
```suggestion
The following providers are supported out-of-the-box by Kimai: 
```

File: _documentation/permission-structure.md [hunk 3]
```diff
@@ -43,14 +43,14 @@ In `sets` we define the "permissions sets" names `ACTIVITY`, `TIMESHEET` and `PR
 In `maps` we now apply the "permission set" called `TIMESHEET` and `ACTIVITY` to the user-role `ROLE_USER` 
 and the three "permission sets" called `TIMESHEET` and `ACTIVITY` and `PROFILE` to the user-role `ROLE_ADMIN`.
 
-At this step the roles have the following permissions:
+At this step, the roles have the following permissions:
```
Comment (kevinpapst) [inline _documentation/permission-structure.md:46]:
```suggestion
At this stage, the roles have the following permissions:
```

File: _documentation/permission-structure.md [hunk 4]
```diff
@@ -43,14 +43,14 @@ In `sets` we define the "permissions sets" names `ACTIVITY`, `TIMESHEET` and `PR
...
 
 - `ROLE_USER` - view_own_timesheet, start_own_timesheet, view_activity, create_activity
 - `ROLE_ADMIN` - view_own_timesheet, start_own_timesheet, view_activity, create_activity, my_profile, show_roles, other_profiles
 
 As last step, the list of "permission names" in the `roles` section will be merged with the list of previously calculated permissions.
 
-At the end the system calculated the final list of permissions:  
+In the end, the system calculated the final list of permissions:
```
Comment (kevinpapst) [inline _documentation/permission-structure.md:53]:
```suggestion
At the end, the system calculated the final list of permissions:  
```

https://www.deepl.com/write also suggests `At`

File: _documentation/plugin-custom-fields.md [hunk 5]
```diff
@@ -42,7 +42,7 @@ If a checkbox is marked as mandatory, the user has to check it in order to submi
 ### Choice-list
 
 "Choice-list" is a different word for "Select-box" or "Drop-down".
-You have to add the entries as comma separated list into the default-value field.
+You have to add the entries as comma-separated list into the default-value field.
```
Comment (kevinpapst) [inline _documentation/plugin-custom-fields.md:45]:
```suggestion
You have to add the entries as comma-separated list in the `Default value` field.
```

File: _documentation/plugin-management.md [hunk 6]
```diff
@@ -14,15 +14,15 @@ Installation should be done with `git clone` or with a copy task.
 Make sure that you copy it into the appropriate subdirectory in `var/plugins/`. 
 For the plugin `CustomCSSBundle` this would be `var/plugins/CustomCSSBundle/` (there should be a file `var/plugins/CustomCSSBundle/CustomCSSBundle.php`).
 
-After the plugin was installed you need to [clear the cache]({% link _documentation/cache.md %}) (see below).
+After the plugin was installed, you need to [clear the cache]({% link _documentation/cache.md %}) (see below).
```
Comment (kevinpapst) [inline _documentation/plugin-management.md:17]:
```suggestion
After installing the plugin, you will need to [clear the cache]({% link _documentation/cache.md %}) (see below).
```

File: _documentation/plugin-management.md [hunk 7]
```diff
@@ -14,15 +14,15 @@ Installation should be done with `git clone` or with a copy task.
...
 
 {% include alert.html icon="fas fa-exclamation" type="warning" alert="Additional installation steps might be required (eg. database creation). Please read the specific plugin docs!" %}
 
 ## Deactivate plugins
 
 You can disable plugins by creating a file called `.disabled` in the plugin directory, e.g. `var/plugins/CustomCSSBundle/.disabled`.
 
-You need to [clear the cache]({% link _documentation/cache.md %}) afterwards (see below). 
+You need to [clear the cache]({% link _documentation/cache.md %}) afterward (see below).
```
Comment (kevinpapst) [inline _documentation/plugin-management.md:25]:
```suggestion
You will then need to [clear the cache]({% link _documentation/cache.md %}) (see below). 
```

File: _documentation/sql-examples.md [hunk 8]
```diff
@@ -66,7 +66,7 @@ So `1,745.4` is one-thousand-seven-hundred-forty-five hours and 24 minutes.
 ### Find entries where hourly rate does not match users hourly rate
 
 Compares timesheet hourly rate and user-preference rate. 
-Does not check configured customer/project/activity rates!
+Do not check configured customer/project/activity rates!
```
Comment (kevinpapst) [inline _documentation/sql-examples.md:69]: The SQL statement does not check ...

File: _documentation/store.md [hunk 9]
```diff
@@ -39,7 +39,7 @@ bundle:
 title: Foo
 type: plugin
 ```
-Also you have to replace `{% raw %}{% include store/keleo-custom-content-bundle.md %}{% endraw %}` with some introduction text for your new item.
+Also, you have to replace `{% raw %}{% include store/keleo-custom-content-bundle.md %}{% endraw %}` with some introduction text for your new item.
```
Comment (kevinpapst) [inline _documentation/store.md:42]:
```suggestion
You also need to replace `{% raw %}{% include store/keleo-custom-content-bundle.md %}{% endraw %}` with some introductory text for your new item.
```

File: _documentation/user-preferences.md [hunk 10]
```diff
@@ -25,7 +25,7 @@ Needs `hourly-rate_own_profile` / `hourly-rate_other_profile` permission to see
 
 ## Timezone
 
-The user-specific timezone is used when recording users timesheet entries. 
+The user-specific timezone is used when recording users' timesheet entries.
```
Comment (kevinpapst) [inline _documentation/user-preferences.md:28]:
```suggestion
The user specific time zone is used when recording user timesheet entries. 
```

File: _documentation/user-preferences.md [hunk 11]
```diff
@@ -38,7 +38,7 @@ Kimai saves times in UTC, including the timezone information, so it can calculat
 
 ## Language
 
-Which language should be used as primary source for translations (if a specific translation is missing for your language, the fallback will always be english).
+Which language should be used as a primary source for translations (if a specific translation is missing for your language, the fallback will always be english).
```
Comment (kevinpapst) [inline _documentation/user-preferences.md:41]:
```suggestion
Which language should be used as the primary source for translations (if a specific translation for your language is missing, the fallback will always be English).
```

**Review by kevinpapst — CHANGES_REQUESTED** [review]
I love your input but 

> any reason why this takes so long to merge?

Many. Even a PR with great intentions means I have to spend my lifetime to check it. No matter the value it brings to the table. I will always do this on my own terms. 

Thanks for the input. I double checked with deepl and it has more suggestions

File: _documentation/emails.md (hunk 1 above)
Comment (simonschaufi) [reply _documentation/emails.md:26]: this is already fixed in another PR.

File: _documentation/plugin-custom-fields.md (hunk 5 above)
Comment (kevinpapst) [reply _documentation/plugin-custom-fields.md]: Yep, I checked the field, it is called like that:
Bildschirmfoto 2023-11-06 um 13 19 26

Comment (simonschaufi) [reply _documentation/plugin-custom-fields.md]: ok, fixed.

**Review by kevinpapst — APPROVED** [review]
Thank you @simonschaufi 👍 🚀

**Pull request merged**